backend/corpora/troonredes/citation/citation.md

```diff
@@ -0,0 +1,53 @@
+## Referencing the dataset
+
+To cite the entire corpus, follow citation guidelines for databases.
+
+We recommend that the collection of transcripts and metadata is attributed to [troonredes.nl](http://troonredes.nl). However, you citation should include that you consulted the version published on I-analyzer.
+
+Note that this references does *not* cover the word models trained on the corpus. See below for recommendations on how to cite the models.
+
+### APA style
+
+> Redactieteam troonredes.nl (2024). *Troonredes* [data set]. I-analyzer. URL: {{ frontend_url }}/search/troonredes
+
+See also [database information in references (APA)](https://apastyle.apa.org/style-grammar-guidelines/references/database-information).
+
+### MLA style
+
+[MLA guidelines](https://style.mla.org/) recommend against citing a database, and recommend [citing each individual work you use](https://style.mla.org/separate-entries-database-works/). If you want to cite the entire corpus nonetheless, we recommend the following format:
+
+> Redactieteam troonredes.nl. "Troonredes". *I-analyzer*, 2024, {{ frontend_url }}/search/troonredes
+
+## Referencing a single speech
+
+To cite a specific speech, follow citation guidelines for orations, and clarify that you used the transcript provided on I-analyzer. We provide examples for a speech below.
+
+### APA style
+
+> Oranje-Nassau, W.F. van (1815). *Troonrede van 16 oktober 1815* [speech transcript]. I-analyzer. URL: {{ frontend_url }}/document/troonredes/1815_troonrede
+
+See also: [citing a speech transcript](https://apastyle.apa.org/style-grammar-guidelines/references/examples/transcript-audiovisual-work-references#1)
+
+## MLA style
+
+> Oranje-Nassau, Willem F. van. "Troonrede van 16 oktober 1815." 16 October 1815. *I-analyzer*, Centre for Digital Humanities (Utrecht University), {{ frontend_url }}/document/troonredes/1815_troonrede. Transcript.
+
+See also: [citing a copy of a speech (MLA)](https://style.mla.org/citing-a-copy-of-a-speech/).
+
+### Other citation styles
+
+Some style guides recommend that a citation of a speech should include both the date and location. The location was not always the same, and is not listed on I-analyzer. You can look it up on [history of Prinsjesdag (website of the Dutch senate)](https://www.eerstekamer.nl/historische_gebeurtenis/geschiedenis_prinsjesdag).
+
+## Referencing word models
+
+While the text dataset of the troonredes is taken from [troonredes.nl](http://troonredes.nl), the word models were trained by the Centre for Digital Humanities at Utrecht University.
+
+If you are the word models interface of this corpus in your research, you can cite it by citing the webpage.
```

Comment:
This is mostly because we don't actually have a citable version of the model available anywhere. If the model was published through zenodo / yoda etc., we could also add that as a reference.

Reply:
After re-training the word models on the current version, we should indeed publish them.